Searching & sorting/Insertion Sort.py

```diff
@@ -0,0 +1,11 @@
+def insertionSort(L):
+    arr=L
+    for i in range(1, len(arr)):
```

Comment:
add comments for explanation and make a separate folder for sorting then try to push

Reply:
Okay , adding the comments.